applications/asset_tracker/src/ui/buzzer.c

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <pwm.h>
+
+#include "ui.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(buzzer, CONFIG_UI_LOG_LEVEL);
+
+#define BUZZER_MIN_FREQUENCY		100
```

Comment:
Should these values be exposed in a kconfig?

Reply:
Will add them